VTUL/iawa
# LIBTD-1816: Add missing files for mirador viewer

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-1816) (:star:)

# What does this Pull Request do? (:star:)
This PR adds several missing directories and their corresponding translation.json as well as debut_dark.png or Mirador viewer as hinted by buildPath config.

# What's the changes? (:star:)

* Add locales/en-US/translation.json into app/assets
* Add locales/en/translation.json into app/assets
* Add debut_dark.png into app/assets/images

# How should this be tested?

* Bring up the application;
* Run as railsapps: bundle exec rake iawa:batch_import["Ms1990_025_Rudoff.index.csv"] to populate some image files under some items;
* When using Mirador viewer to display images, open javascript console, the missing files (translation.json, debut_dark.png) error should disappear.

# Interested parties
Tag (@pmather) interested parties

(:star:) Required fields
